templates/1000-missing.html

```diff
@@ -4,9 +4,10 @@
 {% markdown %}
 # Missing theorems from [Freek Wiedijk's](https://www.cs.ru.nl/~freek/) [1000+ theorems project](https://1000-plus.github.io/)
 These theorems are not yet formalized in Lean (or, these formalisations are not entered in the database yet).
```

Comment:
Happened to notice this:
```suggestion
These theorems are not yet formalized in Lean (or, these formalizations are not entered in the database yet).
```

Could we also display the formalized statements here if they're available?

Reply:
Fixed the typo, and added the statements.